### DISCUSSION TITLE
Very inconsistent result

### DISCUSSION THREAD

**Discussion by RandyBoBandy024** [kaixxx/noScribe#45] [Q&A] [answered]
Hello People, I'm working with noScribe for a while now and lately the results are super inconsistent even tho the audio quality is perfect. Sometimes it removes filler words, sometimes it doesnt. Sometimes it deletes whole sentences, sometimes it doesnt. Sometime it rephrases the senteces and removes unfinished sentences, sometimes it doesnt. Very frustrating. 

My Job is to transcribe one hour long interviews 1:1. Some take me one hour and some take me 3h due to insane inconsistancy. Any solution? If anyone knows how to improve this model, I'm willing to pay

**Comment by kaixxx** [comment, marked as answer]
Hi, nothing changed since the last release back in october (v. 0.4.1), so your results should be the same. But I've also experienced that the recognition of filler words etc. can be inconsistent. 
You can play with the 'prompt' - which is a kind of "example" to show whisper how you want the transcription to look like. If you give a prompt with filler words, unifinished sentences and such, the AI might pick this up. (I had mixed results.) See https://github.com/kaixxx/noScribe#advanced-options The pompt.yml is in the installation directory of noScribe, usually "C:\Program Files (x86)\noScribe" on Windows.
